backend/src/Service/OAICompletionService.php

```diff
@@ -100,4 +101,52 @@ public function countTextsTokens(array $messages): int
 
         return $tokens;
     }
+
+    private function parseMultiToolResponse(CreateResponse $response): CreateResponse
+    {
+        // sometimes the response includes a multi-tool response in message, which we need to parse and execute
+        $prefix = "```multi_tool_use.parallel```\n```json\n";
+        if (!str_starts_with($prefix, $response->choices[0]->message->content)) {
+            return $response;
+        }
+        $responseData = $response->toArray();
+        $stringResponse = trim(
+            str_replace($prefix, '', $responseData['choices'][0]['message']['content']),
+            "` \n\t"
```

Comment:
The `str_starts_with` function is used incorrectly here. The first parameter should be the string to search in (`$response->choices[0]->message->content`), and the second parameter should be the prefix to search for (`$prefix`). This could lead to always failing the prefix check and thus not executing the intended logic inside the `if` block.